server/logger.js

```diff
@@ -0,0 +1,40 @@
+const chalk = require('chalk');
+const ip = require('ip');
+
+const divider = chalk.gray('\n-----------------------------------');
+
+/**
+ * Logger middleware, you can customize it to make messages more personal
+ */
+module.exports = {
+
+  // Called whenever there's an error on the server we want to print
+  onError: err => {
+    console.log(chalk.red(err));
+  },
+
+  // Called when express.js app starts on given port w/o errors
+  onAppStarted: port => {
```

Comment:
Not a fan of calling this `onAppStarted`, I think calling `logger.appStarted` makes more sense. Same thing for `onTunnelStarted` -> `logger.tunnelStarted`